test/built-ins/Temporal/Instant/prototype/round/throws-on-increments-that-do-not-divide-evenly.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.instant.prototype.round
+description: round() throws on increments that do not divide evenly into solar days.
+features: [Temporal]
+---*/
+
+const inst = Temporal.Instant.from("1976-11-18T14:23:30.123456789Z");
```

Comment:
```suggestion
const inst = new Temporal.Instant(0n);
```
(epoch ns doesn't matter here)